java-squid/src/main/java/org/sonar/java/model/statement/ForStatementTreeImpl.java

```diff
@@ -25,30 +25,29 @@
 import org.sonar.java.model.JavaTree;
 import org.sonar.plugins.java.api.tree.ExpressionTree;
 import org.sonar.plugins.java.api.tree.ForStatementTree;
+import org.sonar.plugins.java.api.tree.ListTree;
 import org.sonar.plugins.java.api.tree.StatementTree;
 import org.sonar.plugins.java.api.tree.SyntaxToken;
 import org.sonar.plugins.java.api.tree.Tree;
 import org.sonar.plugins.java.api.tree.TreeVisitor;
 
 import javax.annotation.Nullable;
-
 import java.util.Iterator;
-import java.util.List;
 
 public class ForStatementTreeImpl extends JavaTree implements ForStatementTree {
   private final InternalSyntaxToken forKeyword;
   private final InternalSyntaxToken openParenToken;
-  private final List<StatementTree> initializer;
+  private final ListTree<StatementTree> initializer;
   private final InternalSyntaxToken firstSemicolonToken;
   @Nullable
   private final ExpressionTree condition;
   private final InternalSyntaxToken secondSemicolonToken;
-  private final List<StatementTree> update;
+  private final ListTree<StatementTree> update;
   private final InternalSyntaxToken closeParenToken;
   private final StatementTree statement;
 
-  public ForStatementTreeImpl(InternalSyntaxToken forKeyword, InternalSyntaxToken openParenToken, List<StatementTree> initializer,
-    InternalSyntaxToken firstSemicolonToken, ExpressionTree condition, InternalSyntaxToken secondSemicolonToken, List<StatementTree> update, InternalSyntaxToken closeParenToken,
+  public ForStatementTreeImpl(InternalSyntaxToken forKeyword, InternalSyntaxToken openParenToken, ListTree<StatementTree> initializer,
+    InternalSyntaxToken firstSemicolonToken, ExpressionTree condition, InternalSyntaxToken secondSemicolonToken, ListTree<StatementTree> update, InternalSyntaxToken closeParenToken,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 181 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)